test_regress/t/t_inst_sv.pl

```diff
@@ -17,5 +17,9 @@
     check_finished => 1,
     );
 
+lint(
+    verilator_flags2 => ["--language 1364-2005"]
+    );
+
```

Comment:
This test uses .* so this flag doesn't make sense.  If it was needed to fix some other problem (what?) please instead fix the .v file.

Reply:
Whoops, thanks!